server_admin/topics/authentication/webauthn.adoc

```diff
@@ -4,7 +4,9 @@
 
 {project_name} provides the limited support for https://www.w3.org/TR/webauthn/[W3C Web Authentication (WebAuthn)]. {project_name} works as a WebAuthn's https://www.w3.org/TR/webauthn/#rp-operations[Relying Party (RP)].
 
-IMPORTANT: Please note that WebAuthn support is still in development and not yet complete, so we recommend that you use this feature experimentally. Also, this support's specification and user interfaces may change.
```

Comment:
As I mentioned to you in the private chat, we are in the end going to have WebAuthn enabled by default in the project, but marked as preview in the product. So is it possible to change the docs according to it?

We can possibly keep the previous paragraph "IMPORTANT" just for the project and for the product use the block you added with ":tech_feature_name: WebAuthn" etc?

It is possible to ask somehow in the docs whether you are rendering project and product. Hopefully you are able to figure that and let me know just if not :)

Reply:
Ok, thanks.